pkg/controllers/globalcontext/controller.go

```diff
@@ -88,12 +87,8 @@ func (c *controller) getEntry(name string) (*kyvernov2alpha1.GlobalContextEntry,
 
 func (c *controller) makeStoreEntry(ctx context.Context, gce *kyvernov2alpha1.GlobalContextEntry) (store.Entry, error) {
 	// TODO: should be done at validation time
```

Comment:
@eddycharly Do you mean adding something like this?: https://github.com/kyverno/kyverno/blob/226fa9515a75d9917edb3be01a2fa171be86d458/pkg/webhooks/exception/validate.go#L24-L25

Reply:
Unfortunately yes :(

Reply:
But really, can be done after code freeze or even in patch release imho

Reply:
It's a shame that kubebuilder does not support open api well, this is trivial in open api world !